package.json

```diff
@@ -11,11 +11,26 @@
   },
   "dependencies": {
     "next": "12.0.10",
+    "npm": "^8.4.1",
```

Comment:
Could you remove this as a dependency? We really don't want to expand the potential area of vulnerability if a dependency breaks for whatever reason. We assume every contributor already has Node.js installed which already packages `npm` with it. So, there's pretty much no reason to include this entry in the `package.json` file.

Reply:
ya this got introduced i have removed it